brownie/network/account.py

```diff
@@ -260,7 +262,7 @@ def deploy(
                 {
                     "from": self.address,
                     "value": Wei(amount),
-                    "nonce": self.nonce,
+                    "nonce": nonce if nonce is not None else self.nonce,
```

Comment:
You can reduce this to `nonce or self.nonce` and make it a bit easier to read

Reply:
This will fail tests if manual nonce is set to 0, i think enforcing `None` here is correct

Reply:
Ahh good point.